src/devices/Max7219/MatrixTextWriter.cs

```diff
@@ -0,0 +1,160 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace Iot.Device.Max7219
+{
+
+    /// <summary>
+    /// Matrix text writer can be used to write 
+    /// </summary>
+    public class MatrixTextWriter
```

Comment:
>MatrixTextWriter [](start = 17, length = 16)

I would change this name to something like MatrixGraphics (or just Graphics). the reason is, if we need to add any other graphics functionality (e.g. DrawCircle) we can just add it here. we can have the Font as a property and internally we can use default font if the user didn't specify the font in the constructor (or through the property).

Reply:
ok, renamed to MatrixGraphics

Reply:
Thanks. Do you mind adding a Font property to this class with a getter and setter?

Reply:
Already there: see line 30
```csharp
 public Font Font { get; set; }
```

Reply:
sorry I missed that. thank you.